build/squire-raw.js

```diff
@@ -2912,8 +2919,15 @@ var keyHandlers = {
             if ( previous ) {
                 // If not editable, just delete whole block.
                 if ( !previous.isContentEditable ) {
-                    detach( previous );
-                    return;
+                        if (/svg|use|g|SCRIPT/.test(previous.nodeName) 
```

Comment:
Isn't is better to just test these with case insensitvity? Especially when XML elements' `nodeName` is in the case it's created as (ie. are all our browsers/MathJax working together to always produce lowercase svg elements?)